tests/gcloud-testing-core/src/main/java/com/google/cloud/runtime/jetty/testing/HttpURLUtil.java

```diff
@@ -0,0 +1,57 @@
+package com.google.cloud.runtime.jetty.testing;
+
+import org.eclipse.jetty.toolchain.test.IO;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.StringWriter;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+public final class HttpURLUtil {
```

Comment:
Its specifically designed for the `HttpURLConnection` from the JDK, so it make sense (and conforms to the Google checkstyle rules)

However, I filed it as Issue #49